npm/webpack-dev-server/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const configFiles = [
+  'webpack.config.ts',
+  'webpack.config.js',
+  'webpack.config.mjs',
```

Comment:
Have we tried testing a project that uses `.mjs`? I tried and got an error, but all I did was change an existing project `webpack.config.js` to `webpack.config.mjs` so not sure if it on my side.

<img width="850" alt="Screen Shot 2022-05-31 at 10 04 11 AM" src="https://user-images.githubusercontent.com/25158820/171206216-fe9a9b07-d28a-48d8-9edf-0990c2e2b5cd.png">

Reply:
Wanted to add a +1 here for getting this fixed ASAP. I have run into so much pain over the years with `type: 'module'`, and we should avoid adding to it if possible. Let's open a new issue for this and any other follow-ups, make sure we don't lose track.